packages/wrangler/src/dev.tsx

```diff
@@ -191,39 +190,25 @@ export function devOptions(yargs: CommonYargsArgv) {
 				type: "string",
 				requiresArg: true,
 			})
+			.option("remote", {
+				alias: "r",
+				describe: "Run on a remote machine with access to production resources",
+				type: "boolean",
+				default: false,
+			})
 			.option("local", {
 				alias: "l",
-				describe: "Run on my machine",
+				describe: "Run on my machine (deprecated, now the default)",
 				type: "boolean",
-				default: false, // I bet this will a point of contention. We'll revisit it.
+				deprecated: true,
+				hidden: true,
 			})
 			.option("experimental-local", {
-				describe: "Run on my machine using the Cloudflare Workers runtime",
-				type: "boolean",
-				default: false,
-			})
-			.option("experimental-local-remote-kv", {
 				describe:
-					"Read/write KV data from/to real namespaces on the Cloudflare network",
+					"Run on my machine using the Cloudflare Workers runtime (deprecated, now the default)",
```

Comment:
If it isn't [removed](https://github.com/cloudflare/workers-sdk/pull/3224/files#r1195390755), then

```suggestion
					"Run on my machine using the Cloudflare Workers runtime",
```

since I assume yargs automatically labels deprecated args.